### ISSUE TITLE
Integration Request for AstianGO: A Privacy-Focused Search Engine

### ISSUE THREAD

**Issue by elyerr** [searxng/searxng#3979]
Working URL to the engine
https://astiango.co/api/search?q=gatos&per_page=10&current_page=1
https://astiango.co/api/search/videos?q=gatos&per_page=10&current_page=1
https://astiango.co/api/search/news?q=gatos&per_page=10&current_page=1
https://astiango.co/api/search/images?q=gatos&per_page=10&current_page=1

Why do you want to add this engine?
AstianGO provides a privacy-focused search experience with relevant results. It’s ideal for users who prefer to avoid tracking and data storage, and it offers a comprehensive database of text and multimedia resources.

Features of this engine

    Does not track users and has strict privacy policies
    Fast responses with accurate results    
    Access to multimedia resources (images, videos) and updated news

How can SearXNG fetch the information from this engine?
To fetch information from AstianGO, you can use their public API. Note that requests should include the header `Accept: application/json` to receive responses in JSON format. Here’s an example of a basic request in JavaScript:

javascript example
```
fetch('https://astiango.co/api/search?q=gatos&per_page=10&current_page=1', {
    method: 'GET',
    headers: {
        'Accept': 'application/json'
    }
})
    .then(response => response.json())
    .then(data => console.log(data))
    .catch(error => console.error('Error:', error));
```
The API URL allows various search parameters, and responses are returned in JSON format, making integration straightforward.

Applicable category of this engine

    Web
    Images
    News
    Videos

Additional context
AstianGO is a search engine with an active community and a strong focus on privacy.

**Comment by allendema** [comment]
As this is/was a fork of librex, what are the differences?
https://github.com/hnhx/librex/issues/105#issuecomment-1336113584

Whatever the case might be, in general for searxNG using another proxy metasearch engine can be a nice choice,
but only if they are not too slow and they don't get captcha'd themselvs.

**Comment by Ponchale** [comment]
> As this is/was a fork of librex, what are the differences? [hnhx/librex#105 (comment)](https://github.com/hnhx/librex/issues/105#issuecomment-1336113584)
> 
> Whatever the case might be, in general for searxNG using another proxy metasearch engine can be a nice choice, but only if they are not too slow and they don't get captcha'd themselvs.

No, the first version used LibreX as a base, then advanced to a proprietary development, until reaching the current version, I invite you to try it, and see the speed.
